### DISCUSSION TITLE
GitLab DnD Renovate Runner failing due to whitesource/merge-confidence configuration

### DISCUSSION THREAD

**Discussion by samq-ws** [renovatebot/renovate#14918] [Request Help] [answered]
@rarkins The [GitLab docker in docker renovate runner](https://gitlab.com/renovate-bot/renovate-runner) seems to be throwing an error when it tries to use github>whitesource/merge-confidence:beta from the [template](https://gitlab.com/renovate-bot/renovate-runner/-/blob/main/templates/_common.gitlab-ci.yml). 

GITHUB_COM_TOKEN and RENOVATE_TOKEN with correct permissions have been set up as CI/CD variables within the project
https://gitlab.com/samq-ws/renovate-runner/-/jobs/2284004104

**Comment by rarkins** [comment, marked as answer]
Can you re-run with LOG_LEVEL=debug in that container? One of the reasons it's difficult to understand is because we're seeing none of the debug messages leading up to the error.

**Reply by samq-ws** [reply]
https://gitlab.com/samq-ws/renovate-runner/-/jobs/2288340965 ...very interesting.  We should probably take this offline as it is now working when I replaced the GITHUB_COM_TOKEN with my personal gmail github PAT.
